services/order.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+
+from django.db import transaction
+
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None,
+) -> Order:
+    user = get_user_model().objects.get(username=username)
```

Comment:
The `get()` method will raise a `DoesNotExist` exception if no user is found with the given `username`. Consider handling this exception to prevent potential errors.